### DISCUSSION TITLE
Connect Game test with it?

### DISCUSSION THREAD

**Discussion by MisledWater79** [Mojang/minecraft-editor#12] [Feedback]
I saw the talk on an api for it but I believe that was for more just the editor. I was thinking of maybe also adding gametest with it as well, like having a new module (minecraft/server-editor) that can allow the editor to change values and such in our scripts via like a new tool.

Edit:
Would also be cool to have it so the tab/tool could have the ability to run some of the events without having to actually do it to test scripts. But I don't think this is really needed.

**Comment by dacowan** [comment]
If you could post more specific use cases that would better illustrate your thoughts?

**Reply by MisledWater79** [reply]
A big one could be for debugging, there could be a tab where we can use the module to display to. This would make it so we don't have to use the content log for debugging.

Another could be the ability to change the values of variables in our script.

I said this but make it so we can run events without having to do them for debugging. Or maybe making so we can have fake players so we could see how our scripts runs with multiple people without having alts

**Comment by Xd-pro** [comment]
The @minecraft/server-editor module is already announced for the next update

**Comment by minecraftchloe** [comment]
@Misledwater79 Curious what your thoughts are with the /server-editor module we just released! [aka.ms/EditorAPI](https://aka.ms/EditorAPI) and [Extension Starter Kit](https://github.com/Mojang/minecraft-editor-extension-starter-kit) for help creating an accompanying UI in the Editor.
